app/index.html

```diff
@@ -2,77 +2,51 @@
 <html>
 
 <head>
-	<title>Great Books</title>
-	<meta charset="utf-8" />
-	<link href="kendo/styles/kendo.mobile.all.min.css" rel="stylesheet" />
-	<link href="css/main.css" rel="stylesheet" />
-	<script src="cordova.js"></script>
-	<script src="kendo/js/jquery.min.js"></script>
-	<script src="kendo/js/kendo.mobile.min.js"></script>
-	<script src="js/app.js"></script>
-	<script src="js/details.js"></script>
-	<script src="js/favorites.js"></script>
+    <title>Great Books</title>
```

Comment:
You re-formatted this file but didn't reformat the details.html one. It is good to have identical formatting in all files. If the reformatting here isn't really needed, I suggest reverting it and only applying modifications in the HTML, if any, because now it is hard to see if any HTML was changed (since the entire file appears modified from the re-formatting).